src/frontend/js/components/AddressesManagement/AddressForm.tsx

```diff
@@ -65,6 +65,7 @@ const AddressForm = ({ handleReset, onSubmit, address }: Props) => {
         id="title"
         label={intl.formatMessage(messages.titleInputLabel)}
         error={!!formState.errors.title}
+        message={formState.errors.title?.message}
```

Comment:
If I remember well, this is something I planned to do but I did not do right now because messages are not localized.

Reply:
What do you want to do then? I must admit I'd rather focus on the a11y stuff rather than seeing how to make i18n work with yup :)

I could put this commit aside in order to merge all the other things. Or maybe you want to check the i18n part before?

Reply:
Yes sorry this is more a reminder for me than a request of change. I'll check tomorrow how yup is working right now then I'll come back to you.

Reply:
So, I' checked and yup is not localized by default. So I create a PR #1760 to add localized messages.